proof/refine/ARM/Invariants_H.thy

```diff
@@ -2490,6 +2490,20 @@ lemma cte_wp_at_cases':
                          unless_def when_def bind_def
                   split: kernel_object.splits if_split_asm option.splits
                     del: disjCI)
+apply (rule disjI2)
+
+\<comment> \<open>apply (fastforce simp: field_simps elim: rsubst[where P="\<lambda>x. ksPSpace s x = v" for s v])\<close>
+apply (rule_tac x="0x10" in exI)
+apply (rule_tac x=x6 in exI)
```

Comment:
It looks like there is an additional case in MCS in this proof. Can you comment on what that is? Ideally we want to solve that case without an explicit instantiation.